store/CHANGELOG.md

```diff
@@ -27,6 +27,11 @@ Ref: https://keepachangelog.com/en/1.0.0/
 
 ### Features
 
+- [#15568](https://github.com/cosmos/cosmos-sdk/pull/15568) Migrate the `iavl` to the new key format.
+    - Remov `DeleteVersion`, `DeleteVersions`, `LazyLoadVersionForOverwriting` from `iavl` tree API.
+    - Add `DeleteVersionsTo` and `SaveChangeSet`, since it will keep versions sequentially like `fromVersion` to `toVersion`.
+    - Refactor the pruning manager to use `DeleteVersionsTo`.
+    - Remove `IAVLLazyLoading` option from baseapp.
```

Comment:
This one should be in the main changelog ./CHANGELOG.md instead of under store/CHANGELOG.md